openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -285,6 +286,20 @@ def _wait_until_alive(self):
             raise RuntimeNotReadyError(
                 f'Runtime (ID={self.runtime_id}) is not yet ready. Status: {pod_status}'
             )
+
+        # Wait for pending status
+        while pod_status in ('Pending', 'Running'):
+            time.sleep(2)
```

Comment:
`while X: sleep()` is definitely not a good practice 😬 we need to bail out eventually